### DISCUSSION TITLE
Apt list

### DISCUSSION THREAD

**Discussion by iceman1961** [termux/termux-packages#17574] [Q&A] [answered]
why when I enter "apt list" the PKG list is only g-z there are no packages from a-f?

**Comment by sylirre** [comment, marked as answer]
Terminal output is limited to 2000 lines by default. Repositories have more than 2000 packages. As result, the command output is truncated.

Either change terminal buffer length in `~/.termux/termux.properties` file or use a pager such as `less`.
